src/focus.service.ts

```diff
@@ -587,10 +609,10 @@ export class FocusService {
    * Reset the focus if arcade-machine wanders out of root
    */
   private setDefaultFocus(scrollSpeed: number) {
-    const { root, selected } = this;
-    const allElements = root.querySelectorAll('*');
-    for (let i = 0; i < allElements.length; i += 1) {
-      const potentialElement = <HTMLElement>allElements[i];
+    const { selected } = this;
+    const focusableElems = this.focusRoot.querySelectorAll('*');
```

Comment:
root is the arc root. arcade machine adds arc--selected classes all the way up to the root.
focusRoot is the element whose children are considered for potential focus elements.